cmd/billing.go

```diff
@@ -198,7 +198,7 @@ func redeemCouponCmd(cliCtx *cli.Context) error {
 	if err != nil {
 		switch e := err.(type) {
 		case *discount.PostDiscountsBadRequest:
-			msg := strings.Join(e.Payload.Message, "\n")
+			msg := strings.Join(e.Payload.Messages, "\n")
```

Comment:
Is this intended?

Reply:
Yeah, it caught this typo with the new generated models.